code/game/gamemodes/objective_items.dm

```diff
@@ -139,6 +225,67 @@
 			return 1
 	return 0
 
+//pets

```

Comment:
these should have a common parent

Reply:
also would be cool if you added a lamar objective

Reply:
Yeah! I'll probably will. Thanks for the suggestion.
I do have a question however as I'm pretty new. How do I set the name for the objective with each corresponding animal?

Reply:
objective names can stay different but you can do something like
"[pet.name], alive."
with a pet var on the parent of course